tests/conftest.py

```diff
@@ -63,6 +63,22 @@ def polldevs_conf_with_no_routers(tmp_path):
     yield name
 
 
+@pytest.fixture
+def zino_default_conf(tmp_path):
+    name = tmp_path / "zino.toml"
+    with open(name, "w") as conf:
+        conf.write(
+            """
+            old_events = "old-events"
+            zino_secrets = "secrets"
+            persist_file = "zino-state.json"
+            persist_period = 5
+            conf_check = 1
```

Comment:
.. and polldevs.cf

Reply:
This can be reused to generate the default toml-file on the fly, as a cli-command

Reply:
Not from the tests tree. It would have to be defined somewhere else.

I mean, it would be nice if Zino could generate the example config file from a command line option.  But really, all the values in `zino.toml` should have defaults, so that Zino actually *can* run without a config file present.

The minimum requirement for Zino to run is really the presence of a valid pollfile (`polldevs.cf`) with at least 1 device in it.  Without that, Zino has nothing to do and might as well exit.